src/client/components/AddReservations/AddReservations.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from "react";
+import { useParams } from "react-router";
+import Meals from "../Meals/Meals";
+import "./AddReservations.css";
+import swal from "sweetalert";
+
+export default function AddReservations(props) {
+  // const [reservation, setReservation] = useState("");
+  const [contactName, setContactName] = useState("");
+  const [numberOfGuests, setNumberOfGuests] = useState(1);
+  const [email, setEmail] = useState("");
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [meal, setMeal] = React.useState("");
+  const params = useParams();
+
+  React.useEffect(() => {
+    const mealUrl = `/api/meals/${params.id}`;
+    fetch(mealUrl)
+      .then((response) => response.json())
+      .then((data) => setMeal(data));
+  }, [params.id]);
+
+  console.log("meal", meal);
+
+  const createReservation = (reservation) => {
+    console.log("reservation", reservation);
+    const url = "/api/reservations";
+    if (reservation) {
+      fetch(url, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(reservation),
+      })
+        .then((response) => response.json())
+        .then((data) => {
+          console.log("Success", data);
+        })
+        .catch((error) => {
+          console.log("Error", error);
```

Comment:
Consider adding some notification to the user as well, so he knows his reservation was not successful!